src/Fable.Transforms/Replacements.Util.fs

```diff
@@ -48,6 +48,10 @@ type Helper =
             | Some false | None -> None
         let info = CallInfo.Create(?thisArg=thisArg, args=args, ?sigArgTypes=argTypes, ?genArgs=genArgs, ?memberRef=memberRef, ?isCons=isConstructor)
         Call(callee, info, returnType, loc)
+        
+    static member FSharpRef(com, t, args, ?loc) =
+        let byrefType = DeclaredType({ FullName = Types.byref; Path = CoreAssemblyName "System" }, [ t ])
```

Comment:
@weebs Have you tried using `Types.refCell` intead of `Types.byref`?

Reply:
That's fine, I can address it in another PR. I'll merge it as is and fix it later.

Reply:
Thanks for doing the work to adjust the types @ncave , I tried using `Types.refCell` and ran into TypeScript creating FSharpRef<FSharpRef<>> types for ref constructors. I was able to the tests working by changing one of the makeRef calls but I realized it probably needed more attention and I'm not comfortable touching the various plugins :)

Your fixes helped me understand better how refs were being handled, thanks for the review and working out the remaining pieces!

Reply:
@weebs You're very welcome, thanks for contributing!